cybershell/plugins.py

```
@@ -0,0 +1,52 @@
+from typing import Dict, Any
+from .config import SafetyConfig
+
+@dataclass
+class PluginResult:
+    name: str
+    success: bool
+    details: Dict[str, Any]
+
+class PluginBase:
+    name = 'PluginBase'
+    def __init__(self, config: SafetyConfig):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)